docs/zh/advanced/security.md

```diff
@@ -0,0 +1,253 @@
+# EdgeMesh安全配置
+
+## 目标
+  EdgeMesh服务默认没有开启服务认证，edgemesh-server与edgemesh-agent建立连接时没有经过任何认证
+  如下说明如何通过配置开启EdgeMesh服务之间的安全认证，通过如下的配置，EdgeMesh服务之间会开启身份认证、
+  连接准入、通讯加密
+
+### 依赖
+KubeEdge版本 >= 1.8.2
+
+### edgemesh-server配置
+1. 配置edgemesh-server对应的configmap
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edgemesh-server-cfg
+  namespace: kubeedge
+  labels:
+    k8s-app: kubeedge
+    kubeedge: edgemesh-server
+data:
+  edgemesh-server.yaml: |
+    apiVersion: server.edgemesh.config.kubeedge.io/v1alpha1
+    kind: EdgeMeshServer
+    kubeAPIConfig:
+      burst: 200
+      contentType: application/vnd.kubernetes.protobuf
+      kubeConfig: ""
+      master: ""
+      qps: 100
+    modules:
+      tunnel:
+        enable: true
+        publicIP: "<public-ip>"
+        # 插入如下内容
+        enableSecurity: true
+        tunnelACL:
+          # cloudcore 提供的https服务接口，用于证书签发
+          httpServer: <cloudhub-https-addr>
+```
+2. 配置edgemesh-server对应的deployment
+```yaml
+apiVersion: apps/v1
```

Comment:
can be the security server's deployment config unified with unsecurity case？

Reply:
yes. I have unify the deployment by add the `token` field in the unsecurity case.